app/models/create_join_table_hastags_tweet.rb

```diff
@@ -0,0 +1,2 @@
+class CreateJoinTableHastagsTweet < ApplicationRecord
+end
```

Comment:
This suppose to be join table between tweets and hashtags why there's no relationship here?